.azure-ci.yml

```diff
@@ -7,6 +7,7 @@ trigger:
   branches:
     include:
     - master
+    - release/3.0
```

Comment:
Do we want release/3.0 official builds to be trigger per commit? I don't think we want that. 

cc: @mmitche

Reply:
We do want that for the release branch.

Reply:
I guess the commit ratio in release branch will be pretty low?

Reply:
Very low, my guess is a few a month (a few blits and a few ports)